src/WebJobs.Extensions.DurableTask/Scale/DurableTaskTriggersScaleProvider.cs

```diff
@@ -61,6 +65,26 @@ public DurableTaskTriggersScaleProvider(
                 this.options.HubName);
         }
 
+        private static string? GetConnectionName(IDurabilityProviderFactory durabilityProviderFactory, DurableTaskOptions options)
+        {
+            if (durabilityProviderFactory is AzureStorageDurabilityProviderFactory azureStorageDurabilityProviderFactory)
+            {
+                // First, look for the connection name in the options initialized from the trigger metadata
```

Comment:
The comment is talking about trigger metadata, but I don't see anything trigger specific here. Does the comment need to be updated?

Reply:
@cgillum I'll find a better way to explain this. The `options` parameter contains data (`StorageProvider`) populated from `triggerMetadata` passed to the `DurableTaskTriggersScaleProvider` constructor (line 40).

Reply:
I've updated the comments, I hope it's clear now